node/src/actors/chain_manager/mining.rs

```diff
@@ -916,37 +910,39 @@ fn build_block(
     transaction_fees += reveals_fees;
 
     // Include Mint Transaction by miner
-    let reward = block_reward(epoch) + transaction_fees;
+    let mut reward = block_reward(epoch) + transaction_fees;
 
     // Build Mint Transaction
-    let mint = if split_mint {
-        // The rewards will be split into multiple outputs of the same value as `collateral_minimum`
-        // until the node's set of outputs available for collateralization outnumbers
-        // `data_request_max_retrievals_per_epoch` multiplied by `collateral_coinage`,
-        // i.e. makes sure it always has spare outputs to collateralize.
-
-        let utxos_to_achieve = collateral_age
-            .saturating_mul(u32::from(data_request_max_retrievals_per_epoch))
-            as usize;
-        let utxos_required = utxos_to_achieve.saturating_sub(own_utxos.bigger_than_min_counter());
-
-        MintTransaction::with_split_utxos(
-            epoch,
-            reward,
-            own_pkh,
-            collateral_minimum,
-            utxos_required,
-        )
-    } else {
-        MintTransaction::new(
-            epoch,
-            vec![ValueTransferOutput {
-                pkh: own_pkh,
-                value: reward,
-                time_lock: 0,
-            }],
-        )
-    };
+    let mut vts = vec![];
```

Comment:
Can we extract the mint creation logic into a separate function? Something that takes `mint_reward, external_percentage, external_pkh` and returns a `MintTransaction`. This way it should be easier to test.